workspace/src/main/kotlin/com/cosmotech/workspace/azure/WorkspaceServiceImpl.kt

```diff
@@ -321,6 +324,22 @@ internal class WorkspaceServiceImpl(
         .map { WorkspaceFile(fileName = it) }
   }
 
+  override fun createNamespacedSecretIntoKubernetes(
+      organizationId: String,
+      workspaceId: String,
+      kubernetesDedicatedEventHubSecret: KubernetesDedicatedEventHubSecret
+  ) {
+    val workspaceKey = findWorkspaceById(organizationId, workspaceId).key
+    val secretName = "${organizationId}-${workspaceKey}"
+    val eventBus = csmPlatformProperties.azure?.eventBus!!
+    kubernetesClient.createSecretIntoKubernetes(
+        secretName,
+        kubernetesDedicatedEventHubSecret.namespace,
+        Pair(
+            eventBus.authentication.sharedAccessPolicy?.namespace?.name!!,
```

Comment:
Here it should be a constant for event hub secret (and it is never used)? Like "data" or "secret"

Reply:
In this case, it's a constant value defined in application.yaml and its value is "RootManageSharedAccessKey" as discussed in sprint groomin

Reply:
eventhubnamespace.key